### PR TITLE
adapt-contrib-resources - iOS devices now opening resources in new tab

### PR DESCRIPTION
**Pull request by ryan-lynch**
https://github.com/adaptlearning/adapt_framework/issues/1852

### CODE REVIEW

File: js/adapt-contrib-resourcesView.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ define([
         onResourceClicked: function(event) {
             var data = $(event.currentTarget).data();
 
-            if (!data.forceDownload) {
+            if (!data.forceDownload || Adapt.device.OS.toLowerCase() == 'ios') {
```
Comment (tomgreenfield) [inline js/adapt-contrib-resourcesView.js]: Could you remove the `.toLowerCase()` from here and from line 60 please?

Comment (tomgreenfield) [reply js/adapt-contrib-resourcesView.js]: And switch to strict equality? `===`

Comment (ryan-lynch) [reply js/adapt-contrib-resourcesView.js]: Done!

**Comment by oliverfoster** [comment]
The way resources currently does this is quite odd.
These two lines are duplicate behaviour. The ``window.open`` and ```a.target = "_blank"```;
https://github.com/adaptlearning/adapt-contrib-resources/blob/master/js/adapt-contrib-resourcesView.js#L54
https://github.com/adaptlearning/adapt-contrib-resources/blob/master/js/adapt-contrib-resourcesView.js#L61

https://github.com/adaptlearning/adapt-contrib-resources/blob/master/templates/resources.hbs#L29
^ This is a very weird thing to do with the data. Take it from the javascript world as json, into the dom and then back out again as json into javascript. It breaks the correlation between the data and the behaviour by passing it through the DOM on the way. I would suggest reverting to a ```data-index="0"```:
```javascript
var index = $(event.target).attr("data-index");
var data = this.collection.at(index).toJSON();
```

**Comment by ryan-lynch** [comment]
Could you raise an issue to correct that @oliverfoster

**Comment by oliverfoster** [comment]
https://github.com/adaptlearning/adapt_framework/issues/1853 @ryan-lynch

**Pull request merged**